src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/XsltArgumentList.cs

```diff
@@ -4705,15 +4717,30 @@ public void NodeIter2()
                 xslArg.AddParam("sourceUri", String.Empty, uriSource.ToString());
 
                 xslt.Load(FullFilePath("xsd2cs2.xsl"), new XsltSettings(true, true), new XmlUrlResolver());
-
-                XPathDocument doc = new XPathDocument(FullFilePath("sample.xsd"));
-                StringWriter sw = new StringWriter();
-                xslt.Transform(doc, xslArg, sw);
-                sw.Dispose();
-                return;
             });
 
             Assert.Equal("Compiling JScript/CSharp scripts is not supported", e.InnerException.Message);
```

Comment:
Ha, looks like this is actually done in some places in XML --
 it checks the resource ID, which is OK:
https://github.com/dotnet/corefx/blob/27286da6dfb05cb420e991502ba1654e6e58f752/src/System.Private.Xml/tests/Xslt/XslTransformApi/ExceptionVerifier.cs#L157